downtify/umbrel-app.yml

```
@@ -0,0 +1,23 @@
+manifestVersion: 1
+id: downtify
+category: media
+name: Downtify
+version: "0.2.1"
+tagline: Download Spotify music with album art and metadata
+description: >-
+  With Downtify you can download Spotify musics containing album art, track names, album title and other metadata about the songs.
+  Just copy the Spotify link, whether it's a single song, an album, etc.
+  As soon as your downloads are complete you will be notified!
+developer: Henrique Sebastião
+dependencies: []
+repo: https://github.com/henriquesebastiao/downtify
+port: 8789
+releaseNotes: ""
+support: https://github.com/henriquesebastiao/downtify/issues
+website: https://github.com/henriquesebastiao/downtify
+gallery: []
+path: ""
+defaultUsername: ""
+defaultPassword: ""
+submitter: Henrique Sebastião
+submission: https://github.com/getumbrel/umbrel-apps/pull/2177
```

Add a final newline.
```suggestion
submission: https://github.com/getumbrel/umbrel-apps/pull/2177

```